js/languages/en-US.json

```diff
@@ -2117,5 +2119,235 @@
+    "ko": "Korean",
+    "ko-KR": "Korean (Korea)",
+    "kok": "Konkani",
+    "kok-IN": "Konkani (India)",
+    "ky": "Kyrgyz",
+    "ky-KG": "Kyrgyz (Kyrgyzstan)",
+    "lt": "Lithuanian",
+    "lt-LT": "Lithuanian (Lithuania)",
+    "lv": "Latvian",
+    "lv-LV": "Latvian (Latvia)",
+    "mi": "Maori",
+    "mi-NZ": "Maori (New Zealand)",
+    "mk": "FYRO Macedonian",
+    "mk-MK": "FYRO Macedonian (Former Yugoslav Republic of Macedonia)",
+    "mn": "Mongolian",
+    "mn-MN": "Mongolian (Mongolia)",
+    "mr": "Marathi",
+    "mr-IN": "Marathi (India)",
+    "ms": "Malay",
+    "ms-BN": "Malay (Brunei Darussalam)",
+    "ms-MY": "Malay (Malaysia)",
+    "mt": "Maltese",
+    "mt-MT": "Maltese (Malta)",
+    "nb": "Norwegian (Bokm?l)",
+    "nb-NO": "Norwegian (Bokm?l) (Norway)",
+    "nl": "Dutch",
+    "nl-BE": "Dutch (Belgium)",
+    "nl-NL": "Dutch (Netherlands)",
+    "nn-NO": "Norwegian (Nynorsk) (Norway)",
+    "ns": "Northern Sotho",
+    "ns-ZA": "Northern Sotho (South Africa)",
+    "pa": "Punjabi",
+    "pa-IN": "Punjabi (India)",
+    "pl": "Polish",
+    "pl-PL": "Polish (Poland)",
+    "ps": "Pashto",
+    "ps-AR": "Pashto (Afghanistan)",
+    "pt": "Portuguese",
+    "pt-BR": "Portuguese (Brazil)",
+    "pt-PT": "Portuguese (Portugal)",
+    "qu": "Quechua",
+    "qu-BO": "Quechua (Bolivia)",
+    "qu-EC": "Quechua (Ecuador)",
+    "qu-PE": "Quechua (Peru)",
+    "ro": "Romanian",
+    "ro-RO": "Romanian (Romania)",
+    "ru": "Russian",
+    "ru-RU": "Russian (Russia)",
+    "sa": "Sanskrit",
+    "sa-IN": "Sanskrit (India)",
+    "se": "Sami (Northern)",
+    "se-FI": "Sami (Inari) (Finland)",
+    "se-NO": "Sami (Southern) (Norway)",
+    "se-SE": "Sami (Southern) (Sweden)",
+    "sk": "Slovak",
+    "sk-SK": "Slovak (Slovakia)",
+    "sl": "Slovenian",
+    "sl-SI": "Slovenian (Slovenia)",
+    "sq": "Albanian",
+    "sq-AL": "Albanian (Albania)",
+    "sr-BA": "Serbian (Cyrillic) (Bosnia and Herzegovina)",
+    "sr-SP": "Serbian (Cyrillic) (Serbia and Montenegro)",
+    "sv": "Swedish",
+    "sv-FI": "Swedish (Finland)",
+    "sv-SE": "Swedish (Sweden)",
+    "sw": "Swahili",
+    "sw-KE": "Swahili (Kenya)",
+    "syr": "Syriac",
+    "syr-SY": "Syriac (Syria)",
+    "ta": "Tamil",
+    "ta-IN": "Tamil (India)",
+    "te": "Telugu",
+    "te-IN": "Telugu (India)",
+    "th": "Thai",
+    "th-TH": "Thai (Thailand)",
+    "tl": "Tagalog",
+    "tl-PH": "Tagalog (Philippines)",
+    "tn": "Tswana",
+    "tn-ZA": "Tswana (South Africa)",
+    "tr": "Turkish",
+    "tr-TR": "Turkish (Turkey)",
+    "tt": "Tatar",
+    "tt-RU": "Tatar (Russia)",
+    "ts": "Tsonga",
+    "uk": "Ukrainian",
+    "uk-UA": "Ukrainian (Ukraine)",
+    "ur": "Urdu",
+    "ur-PK": "Urdu (Islamic Republic of Pakistan)",
+    "uz": "Uzbek (Latin)",
+    "uz-UZ": "Uzbek (Cyrillic) (Uzbekistan)",
+    "vi": "Vietnamese",
+    "vi-VN": "Vietnamese (Viet Nam)",
+    "xh": "Xhosa",
+    "xh-ZA": "Xhosa (South Africa)",
+    "zh": "Chinese",
```

Comment:
@jjeffryes I switched things up a little bit from what we discussed. I know we mentioned we were only going to use the base lang and not include a region. But... I remember from my days of working as a localization engineer that there is a substantial difference between simplified (zh-CN) and traditional chinese (zh-TW) - they may even have slightly different alphabets. So, I don't think we could get away with just using the base language as it might be critical to know which "flavor" of that language you mean.

I ended up basing out list off of this one:
http://www.lingoes.net/en/translator/langcode.htm